BiasAndFairnessModule/run_tests.py

```
@@ -0,0 +1,173 @@
+        metrics = list_metrics()
+        print(f"   ✅ Metric registry working, found {len(metrics)} metrics")
+        if len(metrics) > 0:
+            print(f"   📊 Sample metrics: {metrics[:3]}")
+    except Exception as e:
+        print(f"   ❌ Metric registry failed: {e}")
+        return False
+    
+    # Test 3: Individual metrics
+    print("3. Testing individual metrics...")
+    try:
+        from src.eval_engine.metrics import demographic_parity, equalized_odds
```

Ruff (0.12.2)

`src.eval_engine.metrics.equalized_odds` imported but unused

Remove unused import: `src.eval_engine.metrics.equalized_odds`

(F401)